types/socket.io/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for socket.io 2.1
+// Type definitions for socket.io 2.1.1
```

Comment:
Remove the revision. DefinitelyTyped only uses `major.minor` to track definitions. See the FAQ for details: https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

Reply:
Removed! thanks for pointing out 😄